### PR TITLE
Fixed exception when create a new example

### PR DESCRIPTION
**Pull request by zhangshifeng**

### CODE REVIEW

File: screen/ExampleApp/Example/EditExample.xml [hunk 1]
```diff
@@ -18,7 +18,7 @@ along with this software (see the LICENSE.md file). If not, see
 
     <parameter name="exampleId"/>
 
-    <transition name="newExample"><default-response url="."><parameter name="exampleId" value=""/></default-response></transition>
```
Comment (zhangshifeng) [inline screen/ExampleApp/Example/EditExample.xml:21]: Override no effect

**Comment by jonesde** [comment]
The example application has been neglected for quite a while, and uses patterns that I wouldn't recommend like the create form on the edit screen (the SimpleScreens, HiveMind, and PopCommerce screens all do it very differently with a create form on the find screen). I'm planning to do some larger changes and cleanups to the example app so I won't merge this pull request, but I appreciate you bringing it up and reminding me of this much needed effort... I imagine there are many just getting started with Moqui that refer to the example component.

**Comment by zhangshifeng** [comment]
Ok, I'll close this.
Yes, the example component is very important for getting started, I think so too.

**Comment by jonesde** [comment]
FYI the first set of changes to the FindExample and EditExample screens are now in place, demonstrating more useful functionality and all.

**Pull request closed without merging**